aptos-move/framework/aptos-framework/sources/aptos_account.move

```diff
@@ -12,6 +15,28 @@ module aptos_framework::aptos_account {
     const EACCOUNT_NOT_FOUND: u64 = 1;
     /// Account is not registered to receive APT.
     const EACCOUNT_NOT_REGISTERED_FOR_APT: u64 = 2;
+    /// Account opted out of receiving coins that they did not register to receive.
+    const EACCOUNT_DOES_NOT_ACCEPT_DIRECT_COIN_TRANSFERS: u64 = 3;
+    /// Account opted out of directly receiving NFT tokens.
+    const EACCOUNT_DOES_NOT_ACCEPT_DIRECT_TOKEN_TRANSFERS: u64 = 4;
+
+    /// Configuration for whether an account can receive direct transfers of coins that they have not registered.
+    ///
+    /// By default, this is enabled. Users can opt-out by disabling at any time.
+    struct DirectTransferConfig has key {
+        allow_arbitrary_coin_transfers: bool,
+        update_coin_transfer_events: EventHandle<DirectCoinTransferConfigUpdated>,
+    }
+
+    /// Event emitted when an account's direct coins transfer config is updated.
+    struct DirectCoinTransferConfigUpdated has drop, store {
```

Comment:
do we want to call this DirectCoinTransferConfigUpdateEvent or sth to indicate this is an event?

Reply:
Updated